ruby/read_json_file.rb

```diff
@@ -0,0 +1,10 @@
+# Read JSON file, transform and print to stdout
+require 'json'
+
+json_file = ARGV[0]
+
+people = JSON.parse(File.read(json_file))
+
+people.each do |person|
+  puts "Hello, #{person['age']} year old #{person['first_name']}"
+end
```

Comment:
Do I need this `end` at the last line?

Reply:
`people.each` takes a block which is the (`do ... end`)